### ISSUE TITLE
Errors while adding core contributors

### ISSUE THREAD

**Issue by tgoorden** [opencollective/opencollective#898]
I cannot add a core contributor to a collective. Steps to reproduce:

1. Go to the "members" tab.
2. Use "add another member" and fill out the details for an _existing_ account (haven't tried this with an email address that isn't already in the system, but that doesn't seem the cause anyway).
3. Press "save".

This gives me a `Validation error`, with a Javascript stack trace in the console. Appears to be a client-side issue.

***

URL: https://opencollective.com/burgerlijst/edit#members
Logged in as: thomasgoorden
Error message if any:
```
editCollective error:  {"graphQLErrors":[{"message":"Validation error","locations":[{"line":2,"column":3}],"path":["editCollective"]}],"networkError":null,"message":"GraphQL error: Validation error"}
(anonymous) @ editCollective.js:3
```
and
```
editCollective.js:3 Uncaught (in promise) Error: Validation error
    at t.<anonymous> (https://opencollective.com/_next/b711a04f-2dd0-4386-b734-4df60f417c83/page/editCollective.js:3:15573)
    at r (https://opencollective.com/_next/bb3a99c7ea44191318aeaf504447da49/app.js:23:278618)
    at Generator._invoke (https://opencollective.com/_next/bb3a99c7ea44191318aeaf504447da49/app.js:23:279663)
    at Generator.e.(anonymous function) [as throw] (https://opencollective.com/_next/bb3a99c7ea44191318aeaf504447da49/app.js:23:278797)
    at r (https://opencollective.com/_next/bb3a99c7ea44191318aeaf504447da49/app.js:2:145292)
    at https://opencollective.com/_next/bb3a99c7ea44191318aeaf504447da49/app.js:2:145413
    at <anonymous>
```

Browser: Chrome v 63.0

**Comment by tgoorden** [comment]
Note that it is possible that an earlier problem caused an inconsistent state in the data. During an earlier step I pressed save, while there was an extra "core contributor" form open _without any data_. So two "new contributor" forms were open, but only one was filled out when I pressed save. This gave me a different error. Something with a `name` attribute, but unfortunately I didn't save that.
Also note that during that time I had Ghostery running, which was blocking `polyfill.js`. Turned that off though.

**Comment by piamancini** [comment]
hi @tgoorden I'm not seeing anything strange locally and no members where created for your collective. I can do a test to try and reproduce. you will receive an email for a new member created. disregard it please.

**Comment by piamancini** [comment]
where you trying to add a core contributor or a contributor?

**Comment by tgoorden** [comment]
@piamancini I think it's a validation error of some kind. Checking the network activity, I'm seeing that a `POST` does go out to the GraphQL backend. However, the response to that call is an error:
```
{"errors":[{"message":"Validation error","locations":[{"line":2,"column":3}],"path":["editCollective"]}],"data":{"editCollective":null}}
```

**Comment by piamancini** [comment]
ok. I'm going to try and repro.
Are you trying to add Stef as a core contributor?

**Comment by tgoorden** [comment]
@piamancini Exactly.

**Comment by piamancini** [comment]
worked for me.  what browser are you using?

**Comment by piamancini** [comment]
I added him btw so he has access now

**Comment by tgoorden** [comment]
@piamancini Thanks! Using Chrome v 63 (up to date).

**Comment by piamancini** [comment]
ok, as you are unblocked already, I'll label the issue as a bug and we'll look into it on Friday when we fix bugs.

**Comment by clarete** [comment]
Also worked for me but it could indeed be a case of inconsistent state on the frontend side. I'm doing some more digging to see if there's any low hanging fruit in that tree but might be hard to reproduce without the exact steps :/

**Comment by tgoorden** [comment]
So, it seems I can reproduce the initial issue. If I try to add two "Core Contributors" at the same time, but leave the second one empty (!), I get the following error situation:

![oc-error](https://user-images.githubusercontent.com/2543154/36321680-ed123b32-134a-11e8-9a56-b552578f901d.png)

The Javascript console gives me:
```
editCollective.js:3 >>> editCollective error:  {}
(anonymous) @ editCollective.js:3
```
and
```
Uncaught (in promise) Error: Cannot read property 'name' of undefined
    at t.<anonymous> (https://opencollective.com/_next/6336a64d-b938-4b55-b1b4-5fca9b6b1b7e/page/editCollective.js:3:15573)
    at r (https://opencollective.com/_next/63950f576204bad3d6dfe6b07015d216/app.js:23:278618)
    at Generator._invoke (https://opencollective.com/_next/63950f576204bad3d6dfe6b07015d216/app.js:23:279663)
    at Generator.e.(anonymous function) [as throw] (https://opencollective.com/_next/63950f576204bad3d6dfe6b07015d216/app.js:23:278797)
    at r (https://opencollective.com/_next/63950f576204bad3d6dfe6b07015d216/app.js:2:145292)
    at https://opencollective.com/_next/63950f576204bad3d6dfe6b07015d216/app.js:2:145413
    at <anonymous>
```

**Comment by piamancini** [comment]
@clarete are you still working on this one?

**Comment by clarete** [comment]
Sorry @piamancini, this one totally slipped through the cracks :/

**Comment by clarete** [comment]
I could be wrong but I think Xavier fixed this a few weeks ago on another ticket. I'll close it for now!
